### PR TITLE
Homepage & Menu theming, Logo re-added with new favicons generated

### PR DESCRIPTION
**Pull request by AdamMcFet**

### CODE REVIEW

File: project/sites/cosicani/themes/cosicani_theme/templates/block/favicon.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<html><head><link rel="apple-touch-icon" sizes="180x180" href="/sites/cosicani/themes/cosicani_theme/images/favicons/apple-touch-icon.png">
```
Comment (soda736) [inline project/sites/cosicani/themes/cosicani_theme/templates/block/favicon.html:1]: You need to remove the `<html><head>` from this. I think this has been fixed in Unity_theme so not sure why it is still happening here?

File: project/sites/cosicani/themes/cosicani_theme/templates/layout/page.html.twig [hunk 2]
```diff
@@ -93,29 +93,55 @@
...
+      src="sites/cosicani/themes/cosicani_theme/images/cosica-banner.png"
+      alt="">
+  </div>
   {{ page.front_welcome }}
 
   {% if page.top_first or page.top_second or page.top_third %}
     <div id="top-area" class="clearfix">
       {% if page.top_first %}
-        <div class="column">
+        <div id="home-block-1" class="column">
```
Comment (soda736) [inline project/sites/cosicani/themes/cosicani_theme/templates/layout/page.html.twig:106]: Nothing wrong with doing this but you could reduce specificity by just using the column class
e.g. column:first-child, column:nth-child(2), column:last-child?

**Review by soda736 — CHANGES_REQUESTED** [review]

**Comment by soda736** [comment]
Just the project/sites/cosicani/themes/cosicani_theme/templates/block/favicon.html to fix but could consider the other class/ID change too.

File: project/sites/cosicani/themes/cosicani_theme/templates/block/favicon.html (hunk 1 above)
Comment (AdamMcFet) [reply project/sites/cosicani/themes/cosicani_theme/templates/block/favicon.html]: Done

**Review by soda736 — APPROVED** [review]

**Pull request merged**